es7/2016-07/jul-28.md

```diff
@@ -430,7 +429,7 @@ DD: Could be naming, ie. "fin" for ES3?
 
 BE: promise['finally'](...)
 
-JHD: No conflict re: cancellable promises proposal
+JHD: No conflict re: cancelable promises proposal
```

Comment:
("one L" looks wrong to me, so I was actually *adding* L's until I noticed that the proposal used one.)